test/test_consistency.jl

```diff
@@ -13,4 +15,10 @@ end
 g = VNGraph(5); for i in 2:5 Graphs.add_edge!(g,1,i) end; Graphs.add_edge!(g,2,5)
 @test VNGraphs.graph_chromatic_number(g,0)==3
 
+@testset "Graphs.jl Interface Check for VNGraph" begin
+    vngs = [VNGraph(12), VNGraph(20), VNGraph(5)]
```

Comment:
Could you add a few random graphs too? If I am reading this correctly, currently it is being tested only on empty graphs. Something like `VNGraph(Graphs.whatever_random_graph_generator(...))`

Reply:
Sure, makes sense, thank you. I will do that and keep you updated, might take some time for me as I'm not that familiar with these libs (Graphs, VNGraph and VN in C).

Reply:
Done, please feel free to check.